### ISSUE TITLE
Add options to adjust the message body

### ISSUE THREAD

**Issue by fechnert** [reviewdog/reviewdog#437]
# Description

Currently the message body of reviewdog's annotations consists of static parts either reviewdog itself or the used tool has control of.

# Example

For example, if i'm using [pylint](https://www.pylint.org/), i'm only able to adjust the line which comes from pylint's output via the `--msg-template` flag like this:

```yaml
runner:
  pylint:
    cmd: pylint --msg-template='{path}:{line}:{column}:[**[{msg_id}]**] {msg}' --output-format=parseable <path>
    errorformat:
      - "%f:%l:%c:%m"
```

I would like to add some text after the notification from pylint like links to some documentation or an explaining where this message comes from.

# TL;DR

It would be awesome if there were an option to add some text after the annotation from pylint/reviewdog or even have control over the complete body which reviewdog sends via API to the services ...

## Environment

I'm using the following environment:

- reviewdog v0.9.16
- GitLab (self hosted) 12.5.3
- pylint 2.4.4
- python 3.6.5

**Comment by haya14busa** [comment]
Thank you for the request. I also have the idea in my mind, but I didn't implement it yet and I'm still wondering the right design.

BTW, how about adding a text to `--msg-template` for pylint use cases and parse them as a message (`%m`)? e.g. `--msg-template='{path}:{line}:{column}: {msg} [**[{msg_id}]**](https://link/to/msg/{{msg_id}})'`

Actually, actions-eslint uses this method to add a link to rule page for example.
https://github.com/reviewdog/action-eslint/blob/99be606fd081e210a0574765f8a522f0a5e87072/entrypoint.sh#L16

**Comment by fechnert** [comment]
Thank you @haya14busa for your fast response.

We also did this to add links to our messages, but this way it's not possible to add newlines for cool multi-line markdown messages. I think this is because reviewdog interprets newlines as new messages, which won't fit into the required format.

**Comment by haya14busa** [comment]
errorformat supports multi-line text too. [example](https://reviewdog.github.io/errorformat-playground/?efms=%25A%25f%3A%25l%3A%25c%3A+%25m%0A%25C%25m%0A&text=%2Fpath%2Fto%2Ffile1.py%3A1%3A2%3A+message+line+1%0A%5B%7Bmsg_id%7D%5D%28https%3A%2F%2Flink%2Fto%2Frule%2Fpage%2F%7Bmsg_id_1%7D%29%0A%2Fpath%2Fto%2Ffile1.py%3A14%3A2%3A+message+line+1%0Amessage+line+2%0A%5B%7Bmsg_id%7D%5D%28https%3A%2F%2Flink%2Fto%2Frule%2Fpage%2F%7Bmsg_id_2%7D%29%0A%2Fpath%2Fto%2Ffile1.py%3A14%3A15%3A+message+line+1%0Amessage+line+2%0A%5B%7Bmsg_id%7D%5D%28https%3A%2F%2Flink%2Fto%2Frule%2Fpage%2F%7Bmsg_id_3%7D%29)

**Comment by singhprd** [comment]
Hey 👋 

+1 for something like has been suggested here.

I was looking for a way to both:
- have an option to not send the Raw Output to clean up the messages.
- add an option to add a custom messages to a check. For example if a check has failed, then point to documentation for how to fix the reported error.

It looks like changes would need to be made in here?:
/Users/peter/dev/reviewdog/service/github/githubutils/comment_writer.go:50 `ReportAsGitHubActionsLog`

**Comment by fechnert** [comment]
Because pylint is not able to produce multile output (or i didn't find the correct solution yet), i used `<br>`'s instead:

```yaml
runner:
  pylint:
    cmd: pylint --msg-template='{path}:{line}:{column}:[**[{msg_id}](https://...)**] ({category}, {symbol})<br><br>{msg}' --output-format=parseable ...
    errorformat:
      - "%f:%l:%c:%m"

```
